lib/features/support/docker.rb

```diff
@@ -8,6 +8,16 @@ class << self
     # The default place to look for the docker-compose file
     COMPOSE_FILENAME = 'features/fixtures/docker-compose.yml'
 
+    # @!attribute [w] last_exit_code allows reset of the last_exit_code in hooks
+    attr_writer :last_exit_code
+
+    # Stores the last exit code of any docker-compose commands for later comparison
+    #
+    # @returns [Integer, Nil] The last exit code, nil if no command has been run
+    def last_exit_code
+      @last_exit_code ||= nil
+    end
```

Comment:
This can just be an `attr_accessor` as the default is nil if it's not been set/doesn't exist yet

```suggestion
    attr_accessor :last_exit_code
```

e.g.

```ruby
irb(main):001:0> class Abc
irb(main):002:1>   attr_accessor :xyz
irb(main):003:1> end
=> nil
irb(main):004:0> a = Abc.new
=> #<Abc:0x007fbbd99e4a48>
irb(main):005:0> a.xyz
=> nil
irb(main):006:0> a.xyz = 123
=> 123
irb(main):007:0> a.xyz
=> 123
```